tests/test_util.cpp

```diff
@@ -1799,11 +1846,21 @@ void usage(char** argv )
   printf("    --mass-normalization                      # Mass normalization (kappa (default) / mass / asym-mass)\n");
   printf("    --matpc                                   # Matrix preconditioning type (even-even, odd-odd, even-even-asym, odd-odd-asym) \n");
   printf("    --solve-type                              # The type of solve to do (direct, direct-pc, normop, normop-pc, normerr, normerr-pc) \n");
+  printf("    --solution-type                           # The solution we desire (mat (default), mat-dag-mat, mat-pc, "
+         "mat-pc-dag-mat-pc (default for multi-shift))\n");
```

Comment:
Except for staggered!

Reply:
iirc The `--solve-type` is `direct-pc` and the `--solution-type` is `mat-pc`, has anyone made sure this isn't broken by the default changes? I _think_ `staggered_invert_test` hardcodes a bunch of these things if you request the multishift tests.

Reply:
Yes, staggered hard codes a bunch of things, and has done for ages (outside of this pull).  For reasons I can't recall, staggered_invert_test uses `--test` to set these things.  This does need to be fixed for staggered to make it consistent.  Not on this pull.